tests/unit/src/main/scala/tests/DapTestEnrichments.scala

```diff
@@ -1,27 +1,20 @@
 package tests
-import scala.meta.inputs.Position
 import scala.meta.internal.metals.MetalsEnrichments._
 import scala.meta.io.AbsolutePath
 
 import org.eclipse.lsp4j.debug.Source
-import org.eclipse.lsp4j.debug.SourceBreakpoint
 
 object DapTestEnrichments {
   implicit class DapXtensionAbsolutePath(path: AbsolutePath) {
     def toDAP: Source = {
+      val adaptedPath =
+        if (path.isJarFileSystem) path.toURI.toString
```

Comment:
Simulate how VS Code does things. Doesn't change anything if virtual docs are not used.